packages/evolution-common/src/services/surveyGeneration/CSVHandle.ts

```diff
@@ -0,0 +1,21 @@
+import fs = require('fs');
+import Papa = require('papaparse');
+import { SurveyWidgetConfig } from "../../../../../example/demo_survey/src/config/widgets.config";
```

Comment:
We have a parseCsvFile in chaire-lib-common (or chaire-lib-backend for a server side version) which wraps papaparse for our internal API. Please use this instead